src/components/atoms/ImageInput/ImageInput.tsx

```diff
@@ -92,6 +105,11 @@ const ImageInput: React.FC<ImageInputProps> = ({
         value={imageUrl}
         readOnly
       />
+      {isInputHidden && (
+        <Button onClick={onButtonClick} variant="primary">
+          {btnLabel}
+        </Button>
+      )}
```

Comment:
On a second thought, that's not even a label, that's buttonText... or just text

```suggestion
      {isInputHidden && (
        <ButtonNG onClick={onButtonClick} variant="primary">
          {text}
        </ButtonNG>
      )}
```

Reply:
fixed 65d922a